src/tools/pub/pubspec.md

```diff
@@ -398,6 +403,43 @@ the pattern is considered relative to the package's root directory.
   Support for the `false_secrets` field was added in Dart 2.15.
 {{site.alert.end}}
 
+### Screenshots
+
+Packages may showcase their widgets or other visual elements using screenshots.
```

Comment:
Should we explain where these screenshots will be showcased as this is the section opening? Perhaps, something like:

```suggestion
Packages may showcase their widgets or other visual elements using screenshots
that are displayed on their package page.
```

Reply:
Maybe, the verb is  "can".

Because it's something you control as package author. And these docs are mostly geared towards package authors.

Reply:
You're right, edited the suggestion. Thanks.

Reply:
Done.